src/codegeneration/ClassTransformer.js

```diff
@@ -337,10 +371,15 @@ export class ClassTransformer extends TempVarTransformer{
     return createParenExpression(this.makeStrict_(expression));
   }
 
-  transformPropertyMethodAssignment_(tree, homeObject, internalName) {
+  transformPropertyMethodAssignment_(tree, homeObject, internalName, originalName) {
     var parameterList = this.transformAny(tree.parameterList);
     var body = this.transformSuperInFunctionBody_(
         tree.body, homeObject, internalName);
+
+    if (this.showDebugNames_) {
+      tree.debugName = classMethodDebugName(originalName, methodNameFromToken(tree.name), isStatic);
```

Comment:
`tree.name` is a ParseTree here. Then you should be able to use `tree.getStringValue()`? The only special case you need to do is to see if tree.name is `ComputedPropertyName`.